pygmt/tests/test_datasets_earth_relief.py

```diff
@@ -8,36 +8,59 @@
 from pygmt.exceptions import GMTInvalidInput
 
 
-def test_earth_relief_fails():
+@pytest.mark.parametrize(
+    "data_source",
+    ["igpp", "gebco"],
+)
```

Comment:
This can fit on one line
```suggestion
@pytest.mark.parametrize("data_source", ["igpp", "gebco"])
```